projects/admin/src/app/circulation/item/item.component.ts

```diff
@@ -69,8 +69,8 @@ export class ItemComponent implements OnInit {
    * On init hook
    */
   ngOnInit() {
-    if (this.item && this.item.loan && this.item.loan.pid) {
-      const loanPid = this.item.loan.pid;
+    if (this.item && this.item.loan && this.item.loan.item_pid.value) {
```

Comment:
Should it necessary to add a test on `this.item.load.item_pid` ? Just to be sure

Reply:
Yes you're right.